### PR TITLE
blog: fix minor issues

### PR DESCRIPTION
**Pull request by alreadyburnt**
screen

### CODE REVIEW

**Review by selsta — APPROVED** [review]
nice :)

File: blog/index.html [hunk 1]
```diff
@@ -62,32 +62,31 @@ <h3><a href="{{ post.url }}">{{ post.title }}</a></h3>
...
-
-  {% if paginator.next_page %}
-    <a href="{{ paginator.next_page_path | replace: '//', '/' }}">Next &raquo;</a>
-  {% else %}
-    <span>Next &raquo;</span>
-  {% endif %}
-</p></div>
-{% endif %}
+  </section>
+  <a href="#" class="arrow-up" aria-label="{% t accessibility.arrowup %}" title="{% t accessibility.gotop %}"><i></i></a>
```
Comment (erciccione) [inline blog/index.html:91]: The arrow is not showing up correctly

**Review by erciccione — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
ping @alreadyburnt

**Comment by erciccione** [comment]
ping.

**Comment by erciccione** [comment]
PR recovered with #1689

**Pull request closed without merging**